src/components/tile/tile.stories.mdx

```diff
@@ -0,0 +1,273 @@
+            Test Body One
+        </Content>
+        <Content title="Test Title Two">
+            Test Body Two
+        </Content>
+        <Content title="Test Title Three">
+            Test Body Three
+        </Content>
+        </Tile>
+    </Story>
+</Preview>
+
+## Tile with Custom Widths
+The Tile component can also have custom widths. Any child wrapped by a Tile component can be passed an optional `width` prop - this is a percent value, dictating the width of the child element within the tile. 
+Please note that it will not have any effect if the Tile orientation is set to `vertical`.
+
+<Preview>
+    <Story name ='Custom Widths'>
+        <Tile
+        as="tile"
+        orientation="horizontal"
+        padding="M"
+        pixelWidth={0}
+        width={75}
+        >
+        <Content
+            align="left"
+            as="primary"
+            bodyFullWidth={false}
+            inline={false}
+            title="Test Title One"
+            width={40}
+        >
+            Test Body One
+        </Content>
+        <Content
+            align="left"
+            as="primary"
+            bodyFullWidth={false}
+            inline={false}
+            title="Test Title Two"
+            width={80}
+        >
+            Test Body Two
+        </Content>
+        <Content
+            align="left"
+            as="primary"
+            bodyFullWidth={false}
+            inline={false}
+            title="Test Title Three"
+            width={120}
+        >
+            Test Body Three
+        </Content>
+        </Tile>
+    </Story>
+</Preview>
+
+## Tile with Inline Styling
+The Tile component can also have inline styling applied to the content.
+
+<Preview>
+    <Story name ='with Inline'>
+        <Tile
+        as="tile"
+        orientation="horizontal"
+        padding="M"
+        pixelWidth={0}
+        width={0}
+        >
+        <Content
+            align="left"
+            as="primary"
+            bodyFullWidth={false}
+            inline={true}
+            title="Test Title One"
+            width={80}
+        >
+            Test Body One
+        </Content>
+        <Content
+            align="left"
+            as="primary"
+            bodyFullWidth={false}
+            inline={true}
+            title="Test Title Two"
+            width={80}
+        >
+            Test Body Two
+        </Content>
+        <Content
+            align="left"
+            as="primary"
+            bodyFullWidth={false}
+            inline={true}
+            title="Test Title Three"
+            width={80}
+        >
+            Test Body Three
+        </Content>
+        </Tile>
+    </Story>
+</Preview>
+
+## Props of Tile
+<Props of={ Tile } />
+
+# Tile with Definition List
+The Definition List component should only be used inside of the Tile component. The purpose of the Definition List component is to mimic
+the same tags used in HTML5 but instead they are React elements. 
+
```

Comment:
Sorry to be a pain, and I know I haven't done this on Checkbox (I should have), could you add a line here saying that the following spacing props are available:
image and that they can either be a number (multiple of 8px) or any valid CSS string please?

Reply:
✅